### PR TITLE
fix(I Love Music): rewrite presence

### PR DESCRIPTION
**Pull request by DarkVillager**
## Description 

Rewrote the presence due to it being reported as broken [here](https://discord.com/channels/493130730549805057/1053017115449368576).
## Acknowledgements
- [x] I read the [Presence Guidelines](https://github.com/PreMiD/Presences/blob/main/.github/CONTRIBUTING.md)
- [x] I linted the code by running `yarn format`
- [x] The PR title follows the repo's [commit conventions](https://github.com/PreMiD/Presences/blob/main/.github/COMMIT_CONVENTION.md)

## Screenshots
<details>
<summary> Proof showing the creation/modification is working as expected </summary>

</details>

### CODE REVIEW

File: websites/I/I Love Music/presence.ts [hunk 1]
```diff
@@ -1,228 +1,118 @@
...
-//__________________________________________________________________________________________
 
+async function imgPath(path: string, hostname: string) {
+	if (path) {
+		if (path.includes(hostname)) return `https://${path.replace("//", "")}`;
+		else return `https://${hostname}${path}`;
+	} else return Assets.Logo;
+}
+function capitalizeFirstLetter(string: string) {
+	if (string) {
```
Comment (EncryptedDev) [inline websites/I/I Love Music/presence.ts:30]: Why is this check needed?

**Review by EncryptedDev — CHANGES_REQUESTED** [review]

File: websites/I/I Love Music/presence.ts (hunk 1 above)
Comment (DarkVillager) [reply websites/I/I Love Music/presence.ts:30]: it isn’t

Comment (EncryptedDev) [reply websites/I/I Love Music/presence.ts:30]: Remove it then

Comment (DarkVillager) [reply websites/I/I Love Music/presence.ts:30]: okay on it

Comment (DarkVillager) [reply websites/I/I Love Music/presence.ts:30]: It is needed cuz it may be passed undefined values.

**Review by Bas950 — APPROVED** [review]

**Review by EncryptedDev — APPROVED** [review]

**Pull request merged**